apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.IntegrationTests/InsertTest.cs

```diff
@@ -219,5 +221,23 @@ public void InsertRow_RecordRepeatedField()
                 { "score", score },
                 { "gameStarted", gameStarted }
             };
+
+        [Fact]
+        public void CreateLoadJob_Parquet()
+        {
+            string sourceUri = "gs://cloud-samples-data/bigquery/us-states/us-states.parquet";
```

Comment:
nit-question: I think you are following different guidelines to the ones we follow on the extension? regarding the use of `var`. We use `var` only when the type is evident, so this would be a `var` and some of the following `var`s would be replaced by the actual types.

Reply:
I use var much more liberally in test code than production code... but without any dogma around when it's valid or not. We don't use it in snippets though, as there we're really focused on users who may not know any of the types.